test/packages/parallel/gcp/data_stream/compute/_dev/deploy/tf/compute.tf

```diff
@@ -6,17 +6,19 @@ data "google_compute_image" "default" {
 
 resource "google_compute_instance" "default" {
   name = "elastic-package-system-test-${var.TEST_RUN_ID}"
-  // NOTE: e2 instance type is required to collect instance/memory/balloon/* 
+  // NOTE: e2 instance type is required to collect instance/memory/balloon/*
   // metrics, available only on those instances.
   // https://cloud.google.com/monitoring/api/metrics_gcp
   machine_type = "e2-micro"
   zone         = var.zone
 
   labels = {
     run_id = var.TEST_RUN_ID
-    repo_name = var.REPO_NAME
-    pull_request = var.PULL_REQUEST
-    ci_build_number = var.CI_BUILD_NUMBER
+    Environment = "CI"
+    Owner       = "elastic-package"
+    Branch      = var.BRANCH_NAME
+    Build       = var.BUILD_ID
+    CreatedDate = var.CREATED_DATE
```

Comment:
Could we ensure a consistent naming (snake case)?

Reply:
Labels name are snake case, while vars are _ based. Why?

- I used snake case labels as that's the default names I saw in the examples in https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/resource-tagging#getting-started-with-resource-tags
- I used _ vars as that's easy to read while passing TF_VAR_ env variables

Reply:
I just fixed the `run_id` to be camel_case based